src/v0/destinations/tiktok_ads/data/TikTokTrackV2.json

```diff
@@ -68,17 +68,21 @@
   },
   {
     "destKey": "page.url",
-    "sourceKeys": ["properties.url", "context.page.url"],
+    "sourceKeys": ["properties.context.page.url", "properties.url", "context.page.url"],
```

Comment:
why we are adding in front and not at the back?

Reply:
Synced the mapping with the previous version of tiktok